docs/sdk/src/guides/enable_elastic_scaling_mvp.rs

```diff
@@ -0,0 +1,135 @@
+//! # Enable elastic scaling MVP for a parachain
+//!
+//! **This guide assumes full familiarity with Asynchronous Backing and its terminology, as defined
+//! in <https://wiki.polkadot.network/docs/maintain-guides-async-backing>.
+//! Furthermore, the parachain should have already been upgraded according to the guide.**
+//!
+//! ## Quick introduction to elastic scaling
+//!
+//! [Elastic scaling](https://polkadot.network/blog/elastic-scaling-streamling-growth-on-polkadot)
+//! is a feature that will enable parachains to seamlessly scale up or down the amount of block
+//! validated and backed in a single relay chain block, in order to keep up with demand.
+//!
+//! At present, with Asynchronous Backing enabled, a parachain can only include a block on the relay
+//! chain every 6 seconds, irregardless of how many cores the parachain acquires. Elastic scaling
+//! builds further on the 10x throughput increase of Async Backing, enabling collators to submit up
+//! to 3 parachain blocks per relay chain block, resulting in a further 3x throughput increase.
+//!
+//! ## Current limitations of the MVP
+//!
+//! The full implementation of elastic scaling spans across the entire relay/parachain stack and is
+//! still [work in progress](https://github.com/paritytech/polkadot-sdk/issues/1829).
+//! Below are described the current limitations of the MVP:
+//!
+//! 1. **A parachain can use at most 3 cores at a time.** This limitation stems from the fact that
+//!    every parablock has an execution timeout of 2 seconds and the relay chain block authoring
+//!    takes 6 seconds. Therefore, assuming parablock authoring is sequential, a collator only has
+//!    enough time to build 3 candidates in a relay chain slot.
```

Comment:
yeah I created this guide assuming parachains that want to use multiple cores would do so to achieve higher throughput, but it can be also used to achieve lower latency (at least to inclusion in a candidate). I'll rephrase

Reply:
> higher throughput,

can also mean more data.